diff --git a/src/core/android/SDL_android.c b/src/core/android/SDL_android.c
index 86f272e403..c39f2ff3b4 100644
--- a/src/core/android/SDL_android.c
+++ b/src/core/android/SDL_android.c
@@ -513,24 +513,24 @@ static void register_methods(JNIEnv *env, const char *classname, JNINativeMethod
 }

 /* Library init */
-JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved)
-{
-    JNIEnv *env = NULL;
+// JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved)
+// {
+//     JNIEnv *env = NULL;

-    mJavaVM = vm;
+//     mJavaVM = vm;

-    if ((*mJavaVM)->GetEnv(mJavaVM, (void **)&env, JNI_VERSION_1_4) != JNI_OK) {
-        __android_log_print(ANDROID_LOG_ERROR, "SDL", "Failed to get JNI Env");
-        return JNI_VERSION_1_4;
-    }
+//     if ((*mJavaVM)->GetEnv(mJavaVM, (void **)&env, JNI_VERSION_1_4) != JNI_OK) {
+//         __android_log_print(ANDROID_LOG_ERROR, "SDL", "Failed to get JNI Env");
+//         return JNI_VERSION_1_4;
+//     }

-    register_methods(env, "org/libsdl/app/SDLActivity", SDLActivity_tab, SDL_arraysize(SDLActivity_tab));
-    register_methods(env, "org/libsdl/app/SDLInputConnection", SDLInputConnection_tab, SDL_arraysize(SDLInputConnection_tab));
-    register_methods(env, "org/libsdl/app/SDLAudioManager", SDLAudioManager_tab, SDL_arraysize(SDLAudioManager_tab));
-    register_methods(env, "org/libsdl/app/SDLControllerManager", SDLControllerManager_tab, SDL_arraysize(SDLControllerManager_tab));
+//     register_methods(env, "org/libsdl/app/SDLActivity", SDLActivity_tab, SDL_arraysize(SDLActivity_tab));
+//     register_methods(env, "org/libsdl/app/SDLInputConnection", SDLInputConnection_tab, SDL_arraysize(SDLInputConnection_tab));
+//     register_methods(env, "org/libsdl/app/SDLAudioManager", SDLAudioManager_tab, SDL_arraysize(SDLAudioManager_tab));
+//     register_methods(env, "org/libsdl/app/SDLControllerManager", SDLControllerManager_tab, SDL_arraysize(SDLControllerManager_tab));

-    return JNI_VERSION_1_4;
-}
+//     return JNI_VERSION_1_4;
+// }
 
 void checkJNIReady(void)
 {
