diff --git a/cmake/Modules/CocosBuildHelpers.cmake b/cmake/Modules/CocosBuildHelpers.cmake
index 3758b49f1f..765a9f30ba 100644
--- a/cmake/Modules/CocosBuildHelpers.cmake
+++ b/cmake/Modules/CocosBuildHelpers.cmake
@@ -319,7 +319,7 @@ function(cocos_use_pkg target pkg)
         set(_libs ${${prefix}_LIBRARY})
     endif()
     if(_libs)
-        target_link_libraries(${target} ${_libs})
+        target_link_libraries(${target} PRIVATE ${_libs})
         # message(STATUS "${pkg} libs added to '${target}': ${_libs}")
     endif()
 
diff --git a/cmake/Modules/CocosConfigDepend.cmake b/cmake/Modules/CocosConfigDepend.cmake
index 2850686849..678c885773 100644
--- a/cmake/Modules/CocosConfigDepend.cmake
+++ b/cmake/Modules/CocosConfigDepend.cmake
@@ -90,7 +90,7 @@ endmacro()
 macro(use_cocos2dx_libs_depend target)
     cocos2dx_depend()
     foreach(platform_lib ${PLATFORM_SPECIFIC_LIBS})
-        target_link_libraries(${target} ${platform_lib})
+        target_link_libraries(${target} PRIVATE ${platform_lib})
     endforeach()
 
     if(LINUX)
